docs/hdl.rst

```diff
@@ -2,10 +2,43 @@ Hardware Description Languages
 ------------------------------
 
 The two most common **H**\ ardware **D**\ escription **L**\ anguages are
-Verilog and VHDL (the toolchain we are using only supports Verilog).
+Verilog and VHDL.
+
+.. NOTE:: The pre-built toolchain we are releasing supports Verilog only.
+  However, `GHDL <https://github.com/ghdl>`_ might be used as a VHDL frontend
+  for Yosys. See `ghdl-yosys-plugin <https://github.com/ghdl/ghdl-yosys-plugin>`_.
+
+When writing HDL, a tool called ``yosys`` is used to convert the
+human readable verilog into a netlist representation, this is called
+*synthesis*. Once we have the netlist representation, a tool called
+``nextpnr`` performs an operation called *place and route* (P&R) which
+makes it something that will actually run on the FPGA. This is all
+done for you using the ``Makefiles`` in the subdirectories of ``verilog``
+or ``vhdl``.
+
+A big feature of ``nextpnr`` over its predecessor, is the fact that
```

Comment:
Nobody needs to know about `nextpnr`'s predecessor.

Reply:
Agree. I would reword it to:

> Some P&R tools, such as nextpnr, are timing-driven. This means that a design will be generated with a given clock domain guaranteed to perform fast enough, but sometimes it won't further optimise it.

Still, let's decide where to put this content first.